### ISSUE TITLE
The last line of bindings of a hydra is not displayed in hint window.

### ISSUE THREAD

**Issue by c02y** [abo-abo/hydra#331]
when I defined a hydra which contains multiple bindings, the hint window will not dispaly all the bindings, the last line of the bindings will not be dispalyed.

I noticed this when using Spacemacs, at first, I thought this is the problem of spacemacs, then I switched to vanilla emacs using my own init.el, the problem still exists. Checkout the screenshots.
![hydra](https://user-images.githubusercontent.com/2135996/58933637-87871100-879a-11e9-976e-fdc7fae7b03f.png)
![hydra2](https://user-images.githubusercontent.com/2135996/58933638-881fa780-879a-11e9-9a60-7b669f1aba52.png)
![hydra3](https://user-images.githubusercontent.com/2135996/58933639-881fa780-879a-11e9-9e33-7ec9be319de0.png)

Is this a bug, or I didn't use hydra correctly?

**Comment by abo-abo** [comment]
Please post your `defhydra` code.

**Comment by c02y** [comment]
You see the code in the images, of course, if you need text version, here is one:
~~~
(defhydra hydra-multiple-cursors (:hint nil)
             "
         ^Up^                    ^Down^            ^Other^
----------------------------------------------
[_p_]   Prev            [_n_]   Next      [_l_] Edit lines
[_P_]   Skip_P          [_N_]   Skip_N    [_a_] Mark all
[_M-p_] Unmark_P        [_M-n_] Unmark_N  [_r_] Mark by regexp
[_z_]   Undo            [_Z_]   Redo      [_q_] Quit
"
             ("l" mc/edit-lines :exit t)
             ("a" mc/mark-all-like-this :exit t)
             ("n" mc/mark-next-like-this)
             ("N" mc/skip-to-next-like-this)
             ("M-n" mc/unmark-next-like-this)
             ("p" mc/mark-previous-like-this)
             ("P" mc/skip-to-previous-like-this)
             ("M-p" mc/unmark-previous-like-this)
             ("r" mc/mark-all-in-region-regexp :exit t)
             ("z" undo-tree-undo)
             ("Z" undo-tree-redo)
             ("q" nil))
~~~
This is the code in the last screenshot, if you need others, please just tell me.

**Comment by abo-abo** [comment]
Can't reproduce. Can you reproduce with `make run` from this repository. Just copy your defhydra code and paste it in `*scratch*`. Shows all columns for me.

**Comment by c02y** [comment]
No, I cannot reproduce it using `make run` in hydra repo, I guess my other configs cause this problem.

I'll update this post when I figured it out.

![defhydra](https://user-images.githubusercontent.com/2135996/60146802-91ca8700-97fd-11e9-88ad-d0e86b054b5c.png)

**Comment by c02y** [comment]
**UPDATE"**:

After testing my init.el piece by piece, I figured out that this problem is cause by 
~~~
(add-hook 'org-mode-hook 'org-sticky-header-mode)
~~~
`org-sticky-header-mode` is the main reason, although I don't know why.

**Comment by beetleb** [comment]
I have the same issue, and I do _not_ have org-sticky-header installed. I have not yet tried `make run` or other debugging.

**Comment by beetleb** [comment]
Two years later, I finally decided to deep dive into my config to see what is causing this.

In my case, it was this line which I had added as part of my config for `which-func-mode`:

      (setq-default header-line-format
                 '((which-func-mode ("" which-func-format " "))))

This was increasing the header line by one line/row, which essentially "pushed" the bottom row beyond the "viewport". Removing this resolved the problem.

A few notes:

* This happened only in Windows. The same config runs fine in Linux. @c02y : Did this problem occur in Windows or Linux for you?
* It's not specific to hydra. Indeed, when I would run `C-M-l` (i.e. `reposition-window`), it would also be off by one line. I suspect this is more of an Emacs on Windows problem.
